scripts/benchmark_libsplit.sh

```diff
@@ -11,8 +11,10 @@ RUNS=21
 
 echo "Runtime as IEEE Std 1003.2-1992 (``POSIX.2'')"
 echo "Doing $RUNS runs:"
-for run in $(seq 1 $RUNS); do
+i=0
+while [ $i -lt $RUNS ] ; do
```

Comment:
I'm not sure how this behaves on OpenBSD. It works on FreeBSD but it appears from the man-pages that they have different `/bin/sh` implementations.